app/views/works/_required_form.html.erb

```diff
@@ -16,8 +16,8 @@
 
 <!-- Description (one for now)-->
 <div class="field">
-  <%= render partial: "required_form_field_details", locals: { name: "description", display_name: "Description", 
-                                                       field_detail: "Please enter a brief summary of this item specifically &mdash; not identical to the abstract of a corresponding paper. The focus should be on the nature of the materials constituting the item (scope, purpose, methods, etc.), as opposed to substantive claims made in related publications.",
+  <%= render partial: "required_form_field_details", locals: { name: "description", display_name: "Description",
```

Comment:
This is the typo (mdash was not being processed correctly and rendered as `mdash` rather than `-`.